### PR TITLE
feat: support deploy as static website

### PR DESCRIPTION
**Pull request by zhu-xiaowei**
## Description
1. support deploy as static website
2. optimize ui for mobile display
3. update live stream image url

## General Checklist

- [ ] Added new tests to cover change, if needed
- [ ] Security oriented best practices and standards are followed (e.g. using input sanitization, principle of least privilege, etc)
- [x] Documentation update for the change if required
- [x] PR title conforms to conventional commit style
- [ ] If breaking change, documentation/changelog update with migration instructions

By submitting this pull request, I confirm that my contribution is made under the terms of the MIT-0 license.

### CODE REVIEW

File: retail-web/README.md [hunk 1]
```diff
@@ -43,3 +43,10 @@ If you follow [Deployment Instructions](https://github.com/aws-samples/retail-de
    VITE_USE_LOCAL_DATA=false
    ```
 Then you can execute `npm run dev` to preview retail website with your own API.
+
+## Deploy as static website
+Run the following command to deploy this website to your Amazon S3 bucket as a static website.
+
+```bash
+./deploy.sh s3_bucket_name cloudfront_dist_id
```
Comment (zxkane) [inline retail-web/README.md:51]: How could I get the CloudFront distribution ID?

File: retail-web/src/repositories/localVideoRepository.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ export default {
               "28762499-dd36-4b23-96d3-db3eeeaed548",
               "36ba2aec-a64e-4ac5-911d-a36cbbfa83bf"
             ],
-            "thumb_url": "https://d22w93kkp7ubok.cloudfront.net/images/videos/default_stream0_thumb.png"
+            "thumb_url": "https://d202iz0410uej0.cloudfront.net/images/videos/default_stream0_thumb.png"
```
Comment (zxkane) [inline retail-web/src/repositories/localVideoRepository.js:20]: Who is maintaining this CloudFront distribution?

File: retail-web/README.md (hunk 1 above)
Comment (zhu-xiaowei) [reply retail-web/README.md:51]: added how to find CloudFront distribution ID

File: retail-web/src/repositories/localVideoRepository.js (hunk 2 above)
Comment (zhu-xiaowei) [reply retail-web/src/repositories/localVideoRepository.js:20]: and the live video `play_back_url` managed by the origin repo
https://github.com/aws-samples/retail-demo-store/blob/198e98d84c4cceb9b04fc888c6352b1fdc1581dc/videos/default_streams/default_streams.json#L3

**Review by zxkane — APPROVED** [review]

**Pull request merged**